spec/controllers/print_jobs_controller_spec.rb

```diff
@@ -16,10 +16,7 @@
           label_templates_by_service: 'tube_label_template_1d',
           labels: [{ 'label' => { 'barcode' => '12345', 'test_attr' => 'test' } }],
           labels_sprint: {
-            sprint: {
-              'extra_right_text' => 'some x right text',
-              'extra_left_text' => 'some x left text'
-            }
+            sprint: [{ 'extra_right_text' => 'some x right text', 'extra_left_text' => 'some x left text' }]
           },
```

Comment:
Why is this changed to an array? 
As far as I know,  labels_sprint object provides all attributes in a a single object. The labels object on the other hand has separate objects for main label and lid label for a tube in an array for example.

Reply:
![Screenshot 2024-11-20 at 10 31 01 AM](https://github.com/user-attachments/assets/af47dd2a-6980-4e45-b695-79c23663924e)

When given on a single object, if you have multiple tube labels to be printed, the previous tubes will be overwritten by the last tube; therefore printing only the last tube label.

Reply:
Thanks. So it fixes the issue for Squix printing. That will be still a problem if Toshiba printing is used then. Because the array there is not for multiple labels; it is for sections of a single label.